src/main/scala/treadle/executable/ExpressionCompiler.scala

```diff
@@ -32,6 +34,19 @@ class ExpressionCompiler(
     }
   }
 
+  // Preset registers have references to the async reset
+  private val presetTargetList = presetTargets.map { presetTarget =>
```

Comment:
We should add something like this to `firrtl`. I feel like I have written similar code before.
@seldridge @jackkoenig , do you know if there is already a way to turn a reference target into a path string?